app/api/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path, include
+from . views import get_aliases, get_domains, get_alias_filtered
+urlpatterns = [
+    path('aliases', get_aliases, name="get_aliases"),
```

Comment:
`alias `instead of `aliases`

Reply:
can you confirm the endpoint should have api/alias?

Reply:
It should be /api/v1/alias

Reply:
changed